src/api.js

```diff
@@ -3,7 +3,7 @@ let debug       = require('debug')('app:api');
 let slugid      = require('slugid');
 let assert      = require('assert');
 let _           = require('lodash');
-let base        = require('taskcluster-base');
+let base        = require('taskcluster-lib-api');
```

Comment:
Yes, upgrading from base is tricky, because you don't really know what version of the libraries you're upgrading from.  Typically I've just required the one library I need (in this case, tc-lib-api) and left everything else pointing at base.  That said, upgrading everything is worth a try here, and as long as it passes tests and gets @jonasfj's :+1: I'm more than happy to see it done.